server/scripts/standalone/service-standalone.sh

```diff
@@ -50,7 +49,8 @@ start_service() {
         fi
     fi
 
-    nohup java -Xmx1g -XX:MaxDirectMemorySize=512m -jar ${PATH_TO_JAR} --spring.profiles.active=${PROFILE} >> ${STDOUT_LOG} 2>> ${ERROR_LOG} < /dev/null &
+    echo "Running: java -Xmx1g -XX:MaxDirectMemorySize=512m -jar ${PATH_TO_JAR} --spring.profiles.active=${PROFILE} >> ${STDOUT_LOG} 2>> ${ERROR_LOG} < /dev/null &"
+    java -Xmx1g -XX:MaxDirectMemorySize=512m -jar ${PATH_TO_JAR} --spring.profiles.active=${PROFILE} >> ${STDOUT_LOG} 2>> ${ERROR_LOG} < /dev/null &
```

Comment:
Got "nohup: can't detach from console: Inappropriate ioctl for device"

Recommendation seems to be to remove nohup. The server is still running after the session has ended.